velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -111,9 +113,9 @@ TEST_F(ProbabilityTest, normalCDF) {
 }
 
 TEST_F(ProbabilityTest, binomialCDF) {
-  const auto binomialCDF = [&](std::optional<int64_t> numberOfTrials,
+  const auto binomialCDF = [&](std::optional<int32_t> numberOfTrials,
```

Comment:
Try to avoid repetition of the test logic by doing something like https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/tests/ArrayExceptTest.cpp#L141C15-L141C15 that use a type parameter